azure/durable_functions/models/DurableOrchestrationClient.py

```diff
@@ -71,8 +72,23 @@ async def start_new(self,
         request_url = self._get_start_new_url(
             instance_id=instance_id, orchestration_function_name=orchestration_function_name)
 
+        # Get the current span
+        current_span = trace.get_current_span()
+        span_context = current_span.get_span_context()
+
+        # Get the traceparent and tracestate from the span context
+        trace_id = format(span_context.trace_id, '032x')
+        span_id = format(span_context.span_id, '016x')
+        trace_flags = format(span_context.trace_flags, '02x')
```

Comment:
Assuming that these format strings are specified in the Python OpenTelemetry docs - might be good to link this in a comment for future though